### ISSUE TITLE
"Take photo from all cameras" button takes multiple photos from the same camera.

### ISSUE THREAD

**Issue by johnshaughnessy** [mozilla/hubs#2142]
**Description**
Pressing "x" to take a photo from all the cameras in the room instead takes multiple photos from a single camera.

**To Reproduce**
Spawn two cameras in the room. (This requires two clients.)
Press x.
Two photos should appear, both from the same camera.

**Expected behavior**
The button should take one photo from each camera in the room.

**Comment by gfodor** [comment]
When I wrote this feature originally, it was meant to make it so you could take a picture easily from your camera. Not sure if the "take picture from all cameras" is what we want, I hadn't considered it.

**Comment by johnshaughnessy** [comment]
Oh I see -- the bug may just be that it is not prioritizing the camera that you are holding when there are multiple in the room.

**Comment by misslivirose** [comment]
Yeah, taking pictures from all cameras in the room seems like a bit of an odd feature that is prone to possible griefing actions, I might recommend that the fix includes that element.

**Comment by joshmarinacci** [comment]
Has this been fixed?

**Comment by misslivirose** [comment]
No
